packages/js/components/src/product-fields/fields/basic-select-control/render.tsx

```diff
@@ -21,13 +22,16 @@ const SelectControlField: React.FC< SelectControlFieldProps > = ( {
 	options = [],
 } ) => {
 	return (
-		<SelectControl
-			multiple={ multiple }
-			label={ label }
-			options={ options }
-			onChange={ onChange }
-			value={ value }
-		/>
+		<>
+			{ /* @ts-expect-error wrong type for multiple, should be boolean but explicitly set to true/false */ }
```

Comment:
I only added the `Fragment` for this comment 🤷‍♂️ 
Annoyingly the `SelectControl` type for `multiple` sets it to `true` or `false` right away which is different from `boolean`.
So I ignored the error for now: https://github.com/WordPress/gutenberg/blob/bd7e9c2ab0c41269e22b4eb4cd2b2afe3fe84e66/packages/components/src/select-control/types.ts#L67